json_to_df.py

```diff
@@ -0,0 +1,116 @@
+
```

Comment:
This needs to be generalized to other types of sources. In addition, please encompass the work that needs to be done within a `if __name__ == '__main__':` call. This feels like it should probably use some type of CL execution passing at least two parameters -- a directory value and a source value.